core/src/main/java/org/radargun/config/Stage.java

```diff
@@ -15,7 +15,7 @@
    static final String NO_DEPRECATED_NAME = "__no_deprecated_name__";
    String name() default CLASS_NAME_WITHOUT_STAGE;
    String deprecatedName() default NO_DEPRECATED_NAME;
-   String doc();
+   String doc() default "";
```

Comment:
Not having a default value is intended, in order to force you to write some documention.